tests/drivers/build_all/display/app.overlay

```diff
@@ -402,6 +419,7 @@
 				com-invdir;
 				prechargep = <0x22>;
 			};
+
```

Comment:
Same nit re this newline and a separate commit

Reply:
removed it since other PR has commit.